### PR TITLE
Cannot open slice viewer bug

### PR DESCRIPTION
**Pull request by zjmorgan**
**Description of work.**

There is an issue where slice viewer cannot be opened due to `SCALENORM` type not being specified where the data view throws an exception when getting this attribute from the configuration. The fix is: 

```python
 scale = self.conf.get(SCALENORM, type=str)
```

This was not an issue in the initial release of v6.4.

**To test:**

Run this script before and after the fix. In the current version of main, sliceviewer cannot be opened.

```python
# import mantid algorithms, numpy and matplotlib
from mantid.simpleapi import *
import matplotlib.pyplot as plt
import numpy as np

signal = np.linspace(0,1,1000)
error = np.sqrt(signal)

signal -= 0.5

signal[signal>0.25] = np.nan

ws = CreateMDHistoWorkspace(Dimensionality=3,
                            Extents='-3,3,-10,10,-5,5',
                            SignalInput=signal,
                            ErrorInput=error,
                            NumberOfBins='10,10,10',
                            Names='[H,0,0],[0,K,0],[0,0,L]',
                            Units='r.l.u.,r.l.u.,r.l.u.',
                            Frames='HKL,HKL,HKL')
```

*There is no associated issue.*

*This does not require release notes* because it was introduced in this iteration.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py [hunk 1]
```diff
@@ -553,7 +553,7 @@ def get_default_scale_norm(self):
             return scale
 
         if self.conf.has(SCALENORM):
-            scale = self.conf.get(SCALENORM)
+            scale = self.conf.get(SCALENORM, type=str)
 
         if scale == 'Power' and self.conf.has(POWERSCALE):
             exponent = self.conf.get(POWERSCALE)
```
Comment (robertapplin) [inline qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py:559]: I think this line needs `type=str` as well

**Review by robertapplin — CHANGES_REQUESTED** [review]
Thanks for fixing this. I just spotted one thing and then I'm happy to approve

File: qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py (hunk 1 above)
Comment (zjmorgan) [reply qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py:559]: Do you mean line 556 and line 559?

Comment (robertapplin) [reply qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py]: Yes, there was a [recent change](https://github.com/mantidproject/mantid/pull/34089) to make the `type` mandatory when getting config options. Looks like these were accidentally missed

Comment (zjmorgan) [reply qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py]: Should be all squared up. I added some release notes as well.

Comment (martyngigg) [reply qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py]: Do we know why a test didn't catch this?

Comment (robertapplin) [reply qt/python/mantidqt/mantidqt/widgets/sliceviewer/views/dataview.py]: I think this doesn't need release notes as the problem is not in v6.4

There may be a test missing from `test_sliceviewer_dataview` https://github.com/mantidproject/mantid/blob/main/qt/python/mantidqt/mantidqt/widgets/sliceviewer/test/test_sliceviewer_dataview.py

**Review by robertapplin — APPROVED** [review]
This fix looks good to me 👍

**Pull request merged**